src/selectors/entities/channels.js

```diff
@@ -136,6 +137,23 @@ export const getCurrentChannelStats = createSelector(
     }
 );
 
+export const isCurrentChannelReadOnly = createSelector(
+    (state) => state,
```

Comment:
This doesn't work when we pass in the whole state since that's always changing. You could either break this up into
```
export const isCurrentChannelReadOnly = createSelector(
    getCurrentUser,
    getConfig,
    ...
);
```
Or you could probably just get rid of the createSelector since this isn't computationally expensive